save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/usersettingsview/UserSettingsOrganizationsMenuView.kt

```diff
@@ -0,0 +1,50 @@
+package org.cqfn.save.frontend.components.views.usersettingsview
+
+import org.cqfn.save.frontend.components.basic.cardComponent
+import org.cqfn.save.v1
+
+import react.FC
+import react.dom.*
+import react.fc
+
+@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION")
+class UserSettingsOrganizationsMenuView : UserSettingsView() {
+    override fun renderMenu(): FC<UserSettingsProps> = fc { props ->
+        child(cardComponent(isBordered = false, hasBg = true) {
+            div("d-sm-flex align-items-center justify-content-center mb-4 mt-4") {
+                h1("h3 mb-0 mt-2 text-gray-800") {
+                    +"Organizations"
+                }
+            }
+            val organizationsAndRoles = state.selfOrganizations ?: emptyMap()
+
+            ul(classes = "list-group list-group-flush") {
+                for ((organizationName, avatarAndRole) in organizationsAndRoles) {
+                    val (avatar, role) = avatarAndRole
+                    li("list-group-item") {
+                        div("row justify-content-between align-items-center") {
+                            div("align-items-center ml-3") {
+                                img(classes = "avatar avatar-user width-full border color-bg-default rounded-circle") {
+                                    attrs.src = avatar?.let {
+                                        "/api/$v1/avatar$it"
+                                    }
+                                        ?: run {
```

Comment:
Isn't this the thing that you've fixed in diktat? A `run` wrapper for a single statement?

Reply:
It is, yes
Seems things work wrong.

Reply:
Could you please check and open an issue in diktat?

Reply:
I will add this code snippet into existing issue connected to this rule